README.md

```diff
@@ -25,6 +25,10 @@ You must configure the `REACT_APP_MAPBOX_ACCESS_TOKEN` environment variable in a
 nx serve frontend
 ```
 
+#### Style
+
+**Figma** sketch : `https://www.figma.com/proto/lVX7Lycox3AGixBhhbhQsQ/DatAtlas?page-id=592%3A7135&node-id=600%3A8166&viewport=727%2C1558%2C0.5&scaling=min-zoom&starting-point-node-id=600%3A8044`.
```

Comment:
Could you please remove the extra parameters ?
So we may land on the main page instead :
https://www.figma.com/file/lVX7Lycox3AGixBhhbhQsQ/DatAtlas